packages/foam-vscode/src/features/dataviz.ts

```diff
@@ -20,6 +21,17 @@ const feature: FoamFeature = {
         foam.notes.unstable_removeEventListener(onNoteAdded);
       });
 
+      vscode.window.onDidChangeActiveTextEditor(e => {
+        if (e.document.uri.scheme === "file") {
+          const note = foam.notes.getNoteByURI(e.document.uri.fsPath);
+          if (isSome(note)) {
```

Comment:
Not for this PR, but I'm noticing this isSome is starting to get used heavily. Perhaps we should try make the returns from functions that currently result in an "isSome" a little more explicit (null for nothing, for example). What do you think?

Reply:
IIUC that's what we use it for. e.g. `getNoteByURI` will return `null` if no note was found with that URI.
`isSome` is the way we know that something was returned.
Basically anywhere we would tend to use `!== null` or `!== undefined` we can (and should) use `isSome` instead. The same, in the opposite direction, applies to `isNone`.

Am I understanding your point correctly?

Reply:
Ah, yes you do understand correctly. I misunderstood isSome's usage. Thanks for the explanation!